### ISSUE TITLE
gnuradio-companion not working

### ISSUE THREAD

**Issue by jschiefer** [phase4ground/PlusUltra#1]
`gnuradio-companion` is currently not working. When you try to start it, it complains about PyGtk missing.

**Comment by jschiefer** [comment]
Petalinux 2018.2 is based on the "Rocko" (2.4) release of Yocto. In Yocto 2.1 ("Krogoth"), the [pygtk recipe, which is based on GTK2, was removed as obsolete](https://www.yoctoproject.org/docs/current/mega-manual/mega-manual.html#migration-2.1-removed-recipes). It is not clear whether gnuradio-companion will work with the compatibility layers provided by GTK3. 

In a nutshell, there are three options:
1. Port the [pygtk recipe from Yocto 2.0 ("Jethro")](http://git.openembedded.org/openembedded-core/tree/meta/recipes-devtools/python?h=jethro) to Rocko
2. Make gnuradio-companion work with the up-to-date version of PyGtk.
3. Suggested by @dkozel: Build GNU Radio from the `next` branch, which is in the process of updating many of the dependencies.

I took option 1 for a spin, but so far I am having trouble. It appears that during configure, the script has trouble finding pygobject-2.0, although the bitbake cache says it is there. This might be a problem with the cross-compile, or maybe not.

It is probably time to try option 2 or 3.

(Edited: added option 3)
